### ISSUE TITLE
Сonvert fonts from TTF to WOFF and WOFF2

### ISSUE THREAD

**Issue by Prestapro** [itgalaxy/webfont#13]
Hello,

I'm trying to use the plugin to convert not only SVG icons.
My task is to convert fonts from TTF to WOFF and WOFF2
I have not found a way to do this.

Best regards,
Alexandr

**Comment by evilebottnawi** [comment]
@Prestapro thanks for feedback, accepted.

**Comment by misterbrownlee** [comment]
I would +1 this and will even take a look once I get some 🕐

**Comment by romanran** [comment]
There is ttf2woff in the code already, why not just skip the svg part if there is already ttf?

**Comment by evilebottnawi** [comment]
@romanran don't have time now for this PR, your can be champion this :smile:

**Comment by romanran** [comment]
Yeah I forked it and did it, but the ttf2woff throws some strange error.

**Comment by evilebottnawi** [comment]
@romanran your can send PR and we together see that is happens :+1:

**Comment by sergeymorkovkin** [comment]
Two years have passed. Do you accept PR's for this? How long will it take to merge? Implementing this functionality will change how webfont-webpack-plugin & webfont interact, since in TTF case it will return multiple file of the same type.

**Comment by evilebottnawi** [comment]
@sergeymorkovkin yes, PR welcome, just don't have time on this :disappointed: Anyway feel free to send a PR and when we can add support for webfont-webpack-plugin

**Comment by sergeymorkovkin** [comment]
Alright, there you go: [webfont-webpack-plugin](https://github.com/itgalaxy/webfont-webpack-plugin/pull/133) and [webfont](https://github.com/itgalaxy/webfont/pull/185).

**Comment by sergeymorkovkin** [comment]
One thing I forgot to fix is webfont-webpack-plugin/package.json:53. Apologize for this.

**Comment by evilebottnawi** [comment]
Thanks for PR, i will see this in near future

**Comment by sergeymorkovkin** [comment]
Hello! Is there any chance to get it merged in the near future? Thanks.

**Comment by sergeymorkovkin** [comment]
@evilebottnawi Is anything needed to get it published?

**Comment by sergeymorkovkin** [comment]
A couple of thoughts on what's remaining:

1. Update CLI wrapper to support TTF2WOFF(2)
2. Fix code to match all linters
3. Auto-detect SVG/TTF mode without using a new property

Anything else?

**Comment by jimmyandrade** [comment]
> Two years have passed. Do you accept PR's for this? 

@sergeymorkovkin We apologize for the delay in checking your PR https://github.com/itgalaxy/webfont/pull/185.

> Hello! Is there any chance to get it merged in the near future? Thanks.

@sergeymorkovkin because of some changes we made to the main branch, there were some conflicts between your PR and our code. Could you help us to verify these issues so that we can integrate? Thank you.

**Comment by jimmyandrade** [comment]
Investigating [#13](https://github.com/itgalaxy/webfont/issues/13) on branch `feat/issue-13-ttf-to-webfont`.

**Approach (aligned with discussion):**
- Auto-detect `.ttf` input — no `ttfMode` flag (per @sergeymorkovkin’s note in 2019).
- Reuse existing encoders (`ttf2woff`, `wawoff2`, `ttf2eot`) shared with the SVG pipeline.
- Batch runs expose `result.transcodedFonts`; single input keeps top-level `result.woff` / `result.woff2` for CLI compatibility.

PR link will follow shortly. Issue stays open until the fix ships on npm.

**Comment by jimmyandrade** [comment]
PR opened: https://github.com/itgalaxy/webfont/pull/706
